Lib9c/Arena/ArenaHelper.cs

```diff
@@ -136,5 +137,18 @@ public static int GetRewardCount(int score)
 
             return 1;
         }
+
+        public static FungibleAssetValue GetTicketPrice(
+            ArenaSheet.RoundData roundData,
+            ArenaInformation arenaInformation,
+            Currency currency)
+        {
+            var price = roundData.TicketPrice * 0.01m
+                        + (roundData.AdditionalTicketPrice * 0.01m * arenaInformation.PurchasedTicketCount);
+            var major = (long)price;
+            var miner = (int)((price - major) * 100m);
+            var ticketPrice = new FungibleAssetValue(currency, major, miner);
```

Comment:
we can use `FungibleAssetValue.DivRem` like this. https://github.com/planetarium/lib9c/blob/65f8acdf46a54196f83816d42933804483a3d4ab/Lib9c/Helper/CrystalCalculator.cs#L102